src/features/pastelLoading/PastelLoading.module.css

```diff
@@ -0,0 +1,81 @@
+.loading {
```

Comment:
I think we can get away by using one of loaders from http://jonjaques.github.io/react-loaders/ - the "ball beat" type looks the closest to what you made. Reasons to not have our own custom loader for now is:
1. It's generic - doesn't specified in our design and can be anything
2. Increases complexity - we don't need to write a generic loader if there is an established library of generic loaders with 0 dependencies

Reply:
@br4in3x I have updaed for the loading by removed it